src/main/java/core/basesyntax/service/ParserService.java

```diff
@@ -0,0 +1,60 @@
+package core.basesyntax.service;
+
+import core.basesyntax.exception.FruitException;
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ParserService {
+    private static final String SEPARATOR = ",";
+    private static final int OPERATOR_INDEX = 0;
+    private static final int FRUIT_INDEX = 1;
+    private static final int AMOUNT_INDEX = 2;
+    private static final int COUNT_WORDS = 3;
+    private static final String ANNOTATION = "type,fruit,quantity";
+
+    public List<FruitTransaction> convertToListFruitTransaction(List<String> strings) {
```

Comment:
```suggestion
    public List<FruitTransaction> convertToFruitTransactionList(List<String> strings) {
```
Sounds better. However, I'd recommend renaming the service to 'TransactionParserService' and renaming method to simple 'parse'